oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/common/TableHelper.java

```diff
@@ -124,22 +133,19 @@ public List<String> getTablesForRead(String modelName, long timeBucketStart, lon
      */
     public List<String> getTablesInTimeBucketRange(String modelName, long timeBucketStart, long timeBucketEnd) {
         final var model = TableMetaInfo.get(modelName);
-        final var tableName = getTableName(model);
+        final var rawTableName = getTableName(model);
 
         if (!model.isTimeSeries()) {
-            return Collections.singletonList(tableName);
+            return Collections.singletonList(rawTableName);
         }
 
         timeBucketStart = TimeBucket.getTimeBucket(TimeBucket.getTimestamp(timeBucketStart), DownSampling.Day);
         timeBucketEnd = TimeBucket.getTimeBucket(TimeBucket.getTimestamp(timeBucketEnd), DownSampling.Day);
 
-        final var ttlTimeBucketRange = getTTLTimeBucketRange(model);
-
         return LongStream
             .rangeClosed(timeBucketStart, timeBucketEnd)
             .distinct()
-            .filter(ttlTimeBucketRange::contains)
-            .mapToObj(it -> tableName + "_" + it)
+            .mapToObj(timeBucket -> getTable(rawTableName, timeBucket))
             .collect(toList());
```

Comment:
`HistoryDeleteDAO` uses this method to filter tables that should not be deleted, it turns out this method still filter out the `T + 1` table because of `.filter(ttlTimeBucketRange::contains)`,